src/components/DelegationDashboard/DelegateCard.tsx

```diff
@@ -115,136 +158,273 @@ const DelegateCard = ({ delegate, className, trackNum, disabled }: Props) => {
 					)}
 					{!!delegate?.dataSource?.includes(EDelegateSource.NOVA) && (
 						<div className='flex items-center space-x-1 '>
-							<ImageIcon
-								src='/assets/delegation-tracks/nova-wallet.svg'
-								alt='nova wallet icon'
+							<Image
```

Comment:
just use isMobile keyword to check display instead of hidden classaname and two times initialization of <Image